core/src/main/java/org/frankframework/jdbc/XmlQuerySender.java

```diff
@@ -382,37 +375,8 @@ private Message sql(Connection connection, String query, String type) throws Sen
 		}
 	}
 
-	private Message executeUpdate(Connection connection, String tableName, String query, List<Column> columns) throws SenderException {
+	private Message executeUpdate(Connection connection, String query, List<Column> columns) throws SenderException {
 		try {
-			if ((existBlob(columns) && getDbmsSupport().mustInsertEmptyBlobBeforeData()) || (existClob(columns) && getDbmsSupport().mustInsertEmptyClobBeforeData())) {
-				CallableStatement callableStatement = getCallWithRowIdReturned(connection, query);
-				applyParameters(callableStatement, columns);
-				int ri = 1 + countParameters(columns);
-				callableStatement.registerOutParameter(ri, Types.VARCHAR);
-				int numRowsAffected = callableStatement.executeUpdate();
-				String rowId = callableStatement.getString(ri);
-				log.debug(getLogPrefix() + "returning ROWID [" + rowId + "]");
-
-				for (Column column : columns) {
-					if (column.getType().equalsIgnoreCase(TYPE_BLOB) || column.getType().equalsIgnoreCase(TYPE_CLOB)) {
-						query = "SELECT " + column.getName() + " FROM " + tableName + " WHERE ROWID=?" + " FOR UPDATE";
-						QueryType queryType;
-						if (column.getType().equalsIgnoreCase(TYPE_BLOB)) {
-							queryType = QueryType.UPDATEBLOB;
-						} else {
-							queryType = QueryType.UPDATECLOB;
-						}
-						PreparedStatement statement = getStatement(connection, query, queryType);
-						statement.setString(1, rowId);
-						if (column.getType().equalsIgnoreCase(TYPE_BLOB)) {
-							executeUpdateBlobQuery(statement, new Message(column.getValue()));
-						} else {
-							executeUpdateClobQuery(statement, new Message(column.getValue()));
-						}
-					}
-				}
-				return new Message("<result><rowsupdated>" + numRowsAffected + "</rowsupdated></result>");
-			}
```

Comment:
Removing all this looks scary but it appears no longer needed when we no longer need to insert empty blob / clob before inserting the data.